### PR TITLE
Added `Access-Control-Allow-Method` header for the OPTIONS request 

### PR DESCRIPTION
**Pull request by developeruz**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #9137

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

File: framework/rest/OptionsAction.php [hunk 1]
```diff
@@ -40,5 +40,6 @@ public function run($id = null)
         }
         $options = $id === null ? $this->collectionOptions : $this->resourceOptions;
         Yii::$app->getResponse()->getHeaders()->set('Allow', implode(', ', $options));
+        Yii::$app->getResponse()->getHeaders()->set('Access-Control-Allow-Method', implode(', ', $options));
```
Comment (antonkot) [inline framework/rest/OptionsAction.php]: Should be plural: Methods instead of Method

Comment (SilverFire) [reply framework/rest/OptionsAction.php]: @antonkot good catch. Would you like to submit a PR with fix?

Comment (antonkot) [reply framework/rest/OptionsAction.php]: sure

Comment (antonkot) [reply framework/rest/OptionsAction.php]: @SilverFire Here it is: #15869

**Pull request merged**